core/scripts/sql/texera_ddl.sql

```diff
@@ -19,6 +20,14 @@ CREATE TABLE IF NOT EXISTS user
 ) ENGINE = INNODB,
 -- start auto increment userID from 1 because userID 0 means user not exists
   AUTO_INCREMENT = 1;
+  
+CREATE TABLE `user_dictionary` (
+  `uid` int unsigned NOT NULL,
+  `key` varchar(255) COLLATE utf8mb4_bin NOT NULL,
+  `value` text NOT NULL,
+  PRIMARY KEY (`uid`,`key`),
+  FOREIGN KEY (`uid`) REFERENCES `user` (`uid`) ON DELETE CASCADE
+) ENGINE=InnoDB;
```

Comment:
please reformat to align with existing code blocks.

Reply:
fixed